.github/workflows/package.yml

```diff
@@ -4,17 +4,18 @@ name: package
 on:
   workflow_dispatch:
     inputs:
-      setting:
+      location:
+        # TODO: Home doesn't make sense here?
```

Comment:
Maybe remove this TODO since the default changed from "home" to "clinic" --

btw, you have in the release yml:
> location: ["home", "clinic"]

and no default -- is that intended to be different here?

Reply:
`package.yml` is a manual workflow in which you pick a setting, which is why there's a default. `release.yml` runs both home and clinic automatically! I'll leave the TODO because it should probably be fully deleted on the `package.yml` workflow. I just don't want to touch it while I'm still moving files around